src/Serilog.Sinks.Elasticsearch/LoggerConfigurationElasticSearchExtensions.cs

```diff
@@ -143,6 +143,7 @@ public static class LoggerConfigurationElasticsearchExtensions
         /// <param name="failureSink">Sink to use when Elasticsearch is unable to accept the events. This is optionally and depends on the EmitEventFailure setting.</param>   
         /// <param name="singleEventSizePostingLimit"><see cref="ElasticsearchSinkOptions.SingleEventSizePostingLimit"/>The maximum length of an event allowed to be posted to Elasticsearch.default null</param>
         /// <param name="templateCustomSettings">Add custom elasticsearch settings to the template</param>
+        /// <param name="batchAction">Configures the OpType being used when inserting document in batch. Must be set to create for data streams.</param>
```

Comment:
- [ ] Is `batchAction` a good name or should we call it `bulkAction`?

Reply:
Good question; Batch is used inside the sink, while Bulk is the actual command used to communicate to ES. As this is a setting specific to the Elasticsearch operation, it might be better to call it bulkAction instead.